frontend/assets/javascripts/src/modules/ccpa.js

```diff
@@ -0,0 +1,37 @@
+// ----- Imports ----- //
+
+import { Raven } from 'src/modules/raven';
+
+// ----- Functions ----- //
+
+const GEOCOUNTRY_URL = '/geocountry';
+const US_COUNTRY_CODE = 'US';
+
+let fetchCountry;
+
+export const ccpaEnabled = () => {
+    const useCCPA = true; // set false to switch CCPA off
+
+    if (!useCCPA) {
+        return Promise.resolve(false);
+    }
+
+    if (fetchCountry) {
+        return fetchCountry;
+    }
+
+    fetchCountry = fetch(GEOCOUNTRY_URL).then(response => {
+        if (response.ok) {
+            return response.text();
+        } else {
+            throw new Error('failed to get geocountry');
+        }
+    }).then(responseCountryCode => {
+        return responseCountryCode === US_COUNTRY_CODE;
+    }).catch(err => {
+        Raven.captureException(err);
+        return false;
```

Comment:
will we notice if it is following this code path?  not sure if we want to return false when something goes wrong if we are not going to notice anything

Reply:
I've included the `Raven.captureException(err);` call here so we log these errors in `Sentry`, so we'll have visibility there.